domain-server/src/DomainServer.cpp

```diff
@@ -1677,6 +1686,49 @@ void DomainServer::sendHeartbeatToMetaverse(const QString& networkAddress, const
         domainObject[PUBLIC_SOCKET_ADDRESS_KEY] = networkAddressFromSettings;
     }
 
+    QJsonObject capabilities;
+
+#if defined(WEBRTC_DATA_CHANNELS)
+    QJsonObject webRTCCapabilities;
+    bool isWebRTCEnabled = _settingsManager.valueForKeyPath(WEBRTC_ENABLED_SETTINGS_KEY).toBool();
+    bool isWebRTCWSSEnabled = _settingsManager.valueForKeyPath(WEBRTC_WSS_ENABLED_SETTINGS_KEY).toBool();
+    auto acmeDomainList = _settingsManager.valueOrDefaultValueForKeyPath("acme.certificate_domains").toList();
+    QJsonArray acmeDomains;
+    for (auto&& acmeDomain : acmeDomainList) {
+        acmeDomains.push_back(acmeDomain.toMap()["domain"].toString());
+    }
+    webRTCCapabilities["enabled"] = isWebRTCEnabled;
+    webRTCCapabilities["ssl_enabled"] = isWebRTCWSSEnabled;
+    webRTCCapabilities["ssl_domains"] = acmeDomains;
+    capabilities["webrtc"] = webRTCCapabilities;
+#endif
+
+    QJsonObject prometheusCapabilities;
```

Comment:
Need either a more descriptive name than "prometheus" or a comment explaining what Prometheus is.